docs/setup/configuring-logging.asciidoc

```diff
@@ -0,0 +1,697 @@
+
+[float]
+[[dedicated-loggers]]
+===== Dedicated loggers
+
+**Root**
+
+The `root` logger has a dedicated configuration node since this logger is special and should always exist. By default `root` is configured with `info` level and `default` appender that is also always available. This is the configuration that all custom loggers will use unless they're re-configured explicitly.
+
+For example to see _all_ log messages that fall back on the `root` logger configuration, just add one line to the configuration:
+
+[source,yaml]
+----
+logging.root.level: all
+----
+
+Or disable logging entirely with `off`:
+
+[source,yaml]
+----
+logging.root.level: off
+----
+
+**Metrics Logs**
+
+The `metrics.ops` logger is configured with `debug` level and will automatically output sample system and process information at a regular interval.
+The metrics that are logged are a subset of the data collected and are formatted in the log message as follows:
+
+[options="header"]
+|===
+
+| Ops formatted log property | Location in metrics service | Log units
+
+| memory | process.memory.heap.used_in_bytes | http://numeraljs.com/#format[depends on the value], typically MB or GB
+
+| uptime  | process.uptime_in_millis | HH:mm:ss
+
+| load  | os.load | [ "load for the last 1 min" "load for the last 5 min" "load for the last 15 min"]
+
+| delay | process.event_loop_delay | ms
+|===
+
+The log interval is the same as the interval at which system and process information is refreshed and is configurable under `ops.interval`:
+
+[source,yaml]
+----
+ops.interval: 5000
+----
+
+The minimum interval is 100ms and defaults to 5000ms.
+
+[[request-response-logger]]
+**Request and Response Logs**
+
+The `http.server.response` logger is configured with `debug` level and will automatically output
+data about http requests and responses occurring on the {kib} server.
+The message contains some high-level information, and the corresponding log meta contains the following:
+
+[options="header"]
+|===
+
+| Meta property | Description | Format
+
+| client.ip | IP address of the requesting client | ip
+
+| http.request.method | http verb for the request (uppercase) | string
+
+| http.request.mime_type | (optional) mime as specified in the headers | string
+
+| http.request.referrer | (optional) referrer | string
+
+| http.request.headers | request headers | object
+
+| http.response.body.bytes | (optional) Calculated response payload size in bytes | number
+
+| http.response.status_code | status code returned | number
+
+| http.response.headers | response headers | object
+
+| http.response.responseTime | (optional) Calculated response time in ms | number
+
+| url.path | request path | string
+
+| url.query | (optional) request query string | string
+
+| user_agent.original | raw user-agent string provided in request headers | string
+
+|===
+
+[[logging-appenders]]
+=== Appenders
```

Comment:
Appenders renders as a new page in the logs under the Configure logging section:
<img width="547" alt="configure-logging-sections" src="https://user-images.githubusercontent.com/11909450/142695812-f34b7d3c-f69d-47f9-b49f-2d21fd45f3fe.png">

Reply:
I think we should add `[float]` above the section to avoid creating a new page.

Reply:
I wanted to avoid adding too much content on one page but could go either way.